lib/ansible/modules/network/avi/avi_controllerproperties.py

```diff
@@ -61,13 +64,15 @@
         description:
             - Allowed values are 0-1440.
             - Default value when not specified in API or module is interpreted by Avi Controller as 15.
+        type: int
     api_perf_logging_threshold:
         description:
-            - Threshold to log request timing in portal_performance.log and server-timing response header.
+            - Threshold to log request timing in portal_performance. log and server-timing response header.
```

Comment:
Should this be two sentences? Or was the `.` a typo for `,`? The proposed text looks wrong to me.